### DISCUSSION TITLE
Save Wi-Fi connections

### DISCUSSION THREAD

**Discussion by pedrochaves77** [OnionUI/Onion#850] [Ideas]
I have more than one Wi-Fi router at home. So every time I'm playing at different places I have to connect to different routers and write the passwords over and over again. It would be great if for example the last 5 Wi-Fi connections passwords could be saved automatically.

**Comment by XK9274** [comment]
At the minute this is an issue with MainUI and how they handle new connections (overwrites the last one in wpa_supplicant). It's on my list of things to add when the tools become available but for the time being you can use another app I put together that does accomplish this here: https://github.com/XK9274/better-wifi-miyoo

**Reply by pedrochaves77** [reply]
I'll try this app. Thank you very much!
